Mathlib/FieldTheory/Adjoin.lean

```diff
@@ -42,6 +42,10 @@ def adjoin : IntermediateField F E :=
   { Subfield.closure (Set.range (algebraMap F E) ∪ S) with
     algebraMap_mem' := fun x => Subfield.subset_closure (Or.inl (Set.mem_range_self x)) }
 
+@[simp]
+theorem adjoin_toSubfield :
```

Comment:
as previously

Reply:
In this case it is not working, since the definition of `IntermediateField` extends `Subalgebra` but not `Subfield`. The `@[simps]` is not working at all, and `@[simps!]` creates `IntermediateField.adjoin_carrier` which is unwanted, `@[simps! toSubfield]` is not working either.